skipper.go

```diff
@@ -792,12 +796,17 @@ func Run(o Options) error {
 			return err
 		}
 		proxyParams.OpenTracer = tracer
+		proxyParams.OpenTracingInitialSpan = o.OpenTracingInitialSpan
 	} else {
 		// always have a tracer available, so filter authors can rely on the
 		// existence of a tracer
 		proxyParams.OpenTracer, _ = tracing.LoadTracingPlugin(o.PluginDirs, []string{"noop"})
 	}
 
+	if proxyParams.OpenTracingInitialSpan != "" {
+		proxyParams.OpenTracingInitialSpan = o.OpenTracingInitialSpan
```

Comment:
this should catch the skipper use as library, not?

Reply:
well, not if someone uses the proxy package as a library. In theory, it is possible.